packages/core/src/VDocument.ts

```diff
@@ -137,8 +132,7 @@ export class VDocument {
      * If the selection is collapsed, set the format on the selection so we know
      * in the next insert which format should be used.
      */
-    applyFormat(formatName: string): void {
-        const range = this.selection.range;
+    applyFormat(range = this.selection.range, formatName: string): void {
```

Comment:
I'm surprised Typescript let you do this. So `range` is optional but `formatName` isn't. That means if I want to use this with the default value of `range` I have to call it like this: `applyFormat(undefined, 'bold')` ?

Reply:
Nice catch. I inverted the params for the three commands where I did it wrong.